src/ui/templates/insurance/eligibility/check-your-answers.njk

```diff
@@ -16,22 +18,35 @@
     }
   }) }}
 
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-three-quarters-from-desktop">
 
-  <form method="POST" id="form" novalidate>
-    <input type="hidden" name="_csrf" value="{{ csrfToken }}">
+      <span class="govuk-caption-l" data-cy="heading-caption">{{ CONTENT_STRINGS.HEADING_CAPTION }}</span>
+      <h1 class="govuk-heading-l" data-cy="{{ DATA_CY.HEADING }}">{{ CONTENT_STRINGS.PAGE_TITLE }}</h1>
 
-    <div class="govuk-grid-row">
-      <div class="govuk-grid-column-three-quarters-from-desktop">
+      {{ govukSummaryList({
+        rows: SUMMARY_LIST
+      }) }}
 
-        <span class="govuk-caption-l" data-cy="heading-caption">{{ CONTENT_STRINGS.HEADING_CAPTION }}</span>
-        <h1 class="govuk-heading-l" data-cy="{{ DATA_CY.HEADING }}">{{ CONTENT_STRINGS.PAGE_TITLE }}</h1>
+      {{ govukWarningText({
+        text: CONTENT_STRINGS.WARNING,
+        iconFallbackText: "Warning",
+        attributes: {
+          "data-cy": 'warning'
+        }
+      }) }}
+
+      <p class="govuk-body" data-cy="cannot-change-responses">{{ CONTENT_STRINGS.CANNOT_CHANGE }}</p>
```

Comment:
Maybe could use `body` shared page selector here, for E2E tests.

Have updated things so pages with one line of text is called "body". Saves extra data-cy/page selectors being created